### ISSUE TITLE
[feature]: support other decimal separators for measurements input

### ISSUE THREAD

**Issue by anna-puk** [freesewing/freesewing#4343]
### What is it that you would like to see happen? 🤔

I would like to be able to use a comma as the decimal separator when entering my measurements. Currently, I can type the value using a comma, but it does not get saved until I replace the comma with a period.

### Are you a FreeSewing patron? 😃

Yes, I am a tier-8 patron ❤️  💙 💜

**Comment by joostdecock** [comment]
I've added this to #4071 where we track what remains to be done for the v3 release.
We are no longer implementing new features in the v2 branch (which is what is currently running on FreeSewing.org) because we hope to release v3 in the next coming months, so that's where our focus and attention is now.

I will keep this open until it's implemented in v3. Thanks for reporting it.

**Comment by BenJamesBen** [comment]
Not really a full _l10n_ fix. More of a workaround to cheat the UI into accepting inputted comma characters by simply converting them to period characters.
In `sites/shared/utils.mjs`:
```diff
@@ -145,6 +145,8 @@ export const getCrumbs = (app, slug = false) => {
 export const measurementAsMm = (value, units = 'metric') => {
   if (typeof value === 'number') return value * (units === 'imperial' ? 25.4 : 10)
 
+  value = value.replaceAll(',', '.')
+
   if (value.endsWith('.')) return false
 
   if (units === 'metric') {
```

**Comment by eriese** [comment]
Implemented in #4585
